packages/cli-kit/src/public/node/notifications-system.ts

```diff
@@ -0,0 +1,235 @@
+import {versionSatisfies} from './node-package-manager.js'
+import {renderError, renderInfo, renderWarning} from './ui.js'
+import {getCurrentCommandId} from './global-context.js'
+import {fileExists, readFile} from './fs.js'
+import {outputDebug} from './output.js'
+import {CLI_KIT_VERSION} from '../common/version.js'
+import {NotificationsKey, cacheRetrieveOrRepopulate, getCache, setCache} from '../../private/node/conf-store.js'
+
+const URL = 'https://raw.githubusercontent.com/Shopify/cli/notifications/notifications.json'
```

Comment:
We have to change this URL before merging to:

```suggestion
const URL = 'https://raw.githubusercontent.com/Shopify/cli/main/notifications.json'
```